### PR TITLE
fastavro recipe

### PR DESCRIPTION
**Pull request by tebeka**

### CODE REVIEW

File: recipes/fastavro/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+{% set name = "fastavro" %}
+{% set version = "0.10.2" %}
+{% set sha256 = "9eab8b3afd6e25becd7b43d879d703ed24c3e8cc4719f0c8809fd98ef5032bee" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    path: ..
```
Comment (ericdill) [inline recipes/fastavro/meta.yaml]: don't need to give a `path` key when you specify a url

File: recipes/fastavro/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+about:
+    home: https://github.com/tebeka/{{ name }}
+    license: MIT
+    summary: Fast read/write of AVRO files
+    dev_url: https://github.com/tebeka/{{ name }}
+
+extra:
+    recipe-maintainers:
+        - Miki Tebeka
```
Comment (ericdill) [inline recipes/fastavro/meta.yaml]: This should be your github handle

File: recipes/fastavro/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+{% set version = "0.10.2" %}
+{% set sha256 = "9eab8b3afd6e25becd7b43d879d703ed24c3e8cc4719f0c8809fd98ef5032bee" %}
+
+package:
+    name: {{ name }}
+    version: {{ version }}
+
+source:
+    fn: {{name}}-{{ version }}.tar.gz
+    url: http://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
```
Comment (patricksnape) [inline recipes/fastavro/meta.yaml]: `https`

File: recipes/fastavro/test_recipe.py [hunk 4]
```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env python
```
Comment (patricksnape) [inline recipes/fastavro/test_recipe.py]: Did we need this script?

Comment (ocefpaf) [inline recipes/fastavro/test_recipe.py]: Conda does not need that. But it does not hurt if people wants to run with `./run_test.py` when debugging. (Same goes for the `build.sh` we usually keep the shebang, but conda ignores that.)

Comment (patricksnape) [inline recipes/fastavro/test_recipe.py]: Sure it's just something that seems very general but isn't in every recipe.

**Pull request merged**